Moeification.sh

```diff
@@ -22,6 +22,8 @@ if [[ "$@" == *"--credits"* ]]; then
 - Wallpaper: Minimal K-On by https://www.deviantart.com/lucifer012/art/K-ON-Minimalist-Anime-Wallpaper-664262777
 - Wallpaper: Diary of Our Days at the Breakwater by ?
 - Wallpaper: Diary of Our Days at the Breakwater 2 by ?
+- Wallpaper: Your Name Mitsuha by https://www.deviantart.com/ryuzuy/art/Your-Name-867355784
+- Wallpaper: Your Name 2 by Shinkai Makoto
```

Comment:
This needs to be the same full credit as on the grid. So, in other words, "Shinkai Makoto, CoMix Wave Films".